sqlmesh/core/macros.py

```diff
@@ -284,7 +286,7 @@ def evaluate_macros(
                     return exp.to_identifier(text, quoted=node.quoted or None)
             if node.is_string:
                 text = node.this
-                if has_jinja(text):
+                if render_jinja and has_jinja(text):
```

Comment:
I'm investigating why this is needed. Something looks off– I don't think we should be rendering jinja in the macro evaluator intuitively. I'm guessing some test(s) are using unguarded jinja code, i.e. without `JINJA_*` block tags.